crates/matrix-sdk/src/test_utils.rs

```diff
@@ -105,6 +105,7 @@ pub async fn logged_in_client_with_server() -> (Client, wiremock::MockServer) {
 #[macro_export]
 macro_rules! assert_next_with_timeout {
     ($stream:expr, $timeout_ms:expr) => {{
+        #[allow(unused_imports)]
```

Comment:
Why's that suddenly needed?

Reply:
Without it I get:

```
warning: unused import: `futures_util::StreamExt`
    --> crates/matrix-sdk/src/test_utils.rs:108:13
     |
108  |         use futures_util::StreamExt as _;
     |             ^^^^^^^^^^^^^^^^^^^^^^^
     |
    ::: crates/matrix-sdk/src/encryption/mod.rs:1735:9
     |
1735 |         assert_next_matches_with_timeout!(verification_state, VerificationState::Unknown);
     |         --------------------------------------------------------------------------------- in this macro invocation
```

When I try to use the macro in the new test 🫤 .

Reply:
Hmm, Ok.